tools/google-cloud-support-slackbot/README.md

```diff
@@ -0,0 +1,153 @@
+# ![Google Cloud Support Slackbot](google_cloud_support_buddy_small.png)
```

Comment:
Is the use of such a formatted logo permitted by the Google Cloud logo usage policies? It is common in this kind of policies to designate a safe area in the logo and forbidden add additional elements on top of logo.

Reply:
yeah, I reviewed our guidelines and it is going to have no chance of getting marketing approval. I'll work on getting a replacement icon

Reply:
Swapped for one of our icons for now